### PR TITLE
Feature/321438 update alerts to match the new design

### PR DESCRIPTION
**Pull request by ms-Danny**

### CODE REVIEW

File: components/alert/alert.scss [hunk 1]
```diff
@@ -86,6 +99,18 @@
     }
   }
 
+  &__alert-tag {
```
Comment (mattsskea) [inline components/alert/alert.scss:102]: So we are starting to use these badges/tags more often, should we create a component that we reuse in the solution card and alerts?

One more thing, I think we should name the badges and not tags so "__alert-badge" as this is what they are called in the MP solution and we should probably align

File: components/alert/alert.hbs [hunk 2]
```diff
@@ -210,4 +210,74 @@
...
+								class="msds-btn msds-btn--secondary-dark msds-btn--sm">ok,
+								i get it</button>
+						</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<h2>Dark Theme alert - Solution alert, without a checkbox - With image</h2>
```
Comment (mattsskea) [inline components/alert/alert.hbs:247]: I would suggest making a "light themed alert" that uses the badge/tag and possibly also looking at creating documentation for the alerts if you have time, or create a UST that can be prioritized in an upcoming sprint so we can get MSDS a little more clean

File: components/alert/alert.scss (hunk 1 above)
Comment (ms-Danny) [reply components/alert/alert.scss]: Updated the tag to badge

File: components/alert/alert.hbs (hunk 2 above)
Comment (ms-Danny) [reply components/alert/alert.hbs:284]: Added a light theme solution alert.

ill create a ust for documentation

**Review by fcchiaro83 — APPROVED** [review]

**Pull request merged**